backend/degree/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import path
+
+from degree import views
+from degree.views import DegreeListSearch
+
+urlpatterns = [
+    path("degrees/", views.DegreeList.as_view(), name="degree-list"),
+    path(
+        "search/degrees/",
```

Comment:
I think it might make sense to add semester as a URL parameter. And if so what does the semester represent -- the catalog semester?